src/foam/nanos/theme/Themes.js

```diff
@@ -94,21 +94,24 @@ Later themes:
         var group = x.group;
         if ( user && group ) { // non-null when logged in.
           var group = await user.group$find;
+          var defaultMenu = group && group.defaultMenu;
           while ( group ) {
             var groupTheme = await group.theme$find;
-            if ( groupTheme ) {
-              theme = theme && theme.copyFrom(groupTheme) || groupTheme;
-              if ( !! group.defaultMenu ) {
-                theme.defaultMenu = group.defaultMenu;
-                theme.logoRedirect = group.defaultMenu;
-              }
+            if ( groupTheme && ! foam.util.equals(theme, groupTheme) ) {
+              theme = theme && theme.merge(groupTheme) || groupTheme;
```

Comment:
do you really want to break on one group theme merge.
What if you just let it run through all the parents of group.

Reply:
Breaking on one group theme merge is intended. Your own group theme would contain all the overrides the user should need.